packages/react-theme/etc/react-theme.api.md

```diff
@@ -389,6 +428,21 @@ export type ShadowTokens = {
     shadow64: string;
 };
 
+// @public (undocumented)
+export type SpacingTokens = {
```

Comment:
@miroslavstastny The spacing per the figma has the horizontal and vertical sets of these on the alias sheet.  I assume this means that they are the ones exposed as tokens on the theme and the spacing tokens are just the default seed for both of them.  You can see I use the spacing tokens constant to initialize the horizontal/vertical in the theme.

Do you think this type should be exported?